@here/harp-mapview/lib/image/ImageCache.ts

```diff
@@ -89,28 +89,29 @@ export class ImageCache {
     /**
      * Add an image definition to the global cache. Useful when the image data is already loaded.
      *
-     * @param mapView - Specifiy which {@link MapView} requests the image.
+     * @param owner - Specify which owner ({@link MapViewImageCache}) requests the image.
      * @param url - URL of image.
      * @param imageData - Optional {@link ImageData}containing the image content.
-     * @param htmlElement - Optional containing a HtmlCanvasElement or a HtmlImageElement as conten.
+     * @param htmlElement - Optional containing a HtmlCanvasElement or a HtmlImageElement as
+     * content.
      */
     registerImage(
-        mapView: MapView,
+        owner: any,
         url: string,
         imageData?: ImageData | ImageBitmap,
         htmlElement?: HTMLImageElement | HTMLCanvasElement
     ): ImageItem {
         let imageCacheItem = this.findImageCacheItem(url);
         if (imageCacheItem !== undefined) {
-            if (mapView !== undefined && !imageCacheItem.mapViews.includes(mapView)) {
-                imageCacheItem.mapViews.push(mapView);
+            if (owner !== undefined && !imageCacheItem.owners.includes(owner)) {
```

Comment:
do you need the undefined check? The owner is not optional.

Reply:
removed it